news/2019-07-11-project-loved-week-of-july-14th.md

```diff
@@ -0,0 +1,219 @@
+
+[![](/wiki/shared/news/2019-07-11-project-loved-week-of-july-14th/mania/2-chik-habit.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[The Flashbulb - Chik Habit](https://osu.ppy.sh/beatmapsets/732982#mania) by [Gekido-](https://osu.ppy.sh/users/4693052)  
+210 BPM, 2:34 | [4K] 4.17★, [4K] 4.45★
+
+If you've been playing rhythm games for at least a year or two, you'll know who The Flashbulb is. His early crisp drill-and-bass tunes are hard to forget, and the combination of cinematic elements and IDM in his later works is an approachable yet cathartic listen for many. *Chik Habit* is, in the rhythm game community, one of The Flashbulb's most popular songs. There are multiple charts for *Chik Habit* in osu!mania, StepMania, and Flash Flash Revolution that are mind-numbingly difficult, one of which practically impossible. In this wave, we're nominating a chart for *Chik Habit* that is, while not as freakishly difficult as most charts for *Chik Habit*, a chart that is fair, exciting, complementary, and challenging.
+
+Gekido's *Chik Habit* is a medley of fast broken jumpstreams, light minijack usage, difficult 1/6 bursts (315-360 BPM 1/4s) and short 1/8 bursts all compressed into a short 2 and a half minutes. It was also used as a technical qualifier chart that was used in SOFT3 (one of the biggest ever osu!mania tournaments), and it was praised for being a gratifying chart to play through and for having an apposite prudent approach to the song. 
+
+Looking or playing through the chart, it's hard not to see why: Gekido's tactful patterning of the 1/6 bursts make those bursts feel far more intense than they otherwise would, his LN use emphasised certain parts of the melody well, most of the patterns require a lot of wrist movement (which makes the chart really engaging), and the chart, overall, captures the song well through good layering and patterning without making the chart overly spiky in difficulty. Combine that with a catchy and aggressive song, and you get a chart that's hard to miss. For players who aren't as enthusiastic about LNs, Gekido's Challenge difficulty has no LNs, better layering, and more 1/6 bursts -- give that a whirl instead.
+
+[![](/wiki/shared/news/2019-07-11-project-loved-week-of-july-14th/mania/3-sakurairo-time-capsule.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[\*namirin - Sakurairo Time Capsule](https://osu.ppy.sh/beatmapsets/831039#mania) by [MisterLuka](https://osu.ppy.sh/users/6117525), [eZmmR](https://osu.ppy.sh/users/8647138), [upan](https://osu.ppy.sh/users/4289829) and [kaythen](https://osu.ppy.sh/users/5782153)  
```

Comment:
upan has the wrong link, it should be https://osu.ppy.sh/users/5214397 (restricted)